development_tools/intellij_idea.md

```diff
@@ -62,7 +61,11 @@ delicious gravy to be had. My general advice is to keep an eye out for anything
 a better way; often there is a way to get the IDE to do the boring work for you. Don't get hung up on memorizing all of the
 options, which would be impossible; just learn the features you use often and be vaguely aware of the features you use occasionally (so you can look them up when you need them).
 ### Creating a Program
-Right click on the `src` directory and select `New` | `Java Class`.
+Right click on the `src` directory, select `New` | `Java Class`, select `Class`, and choose a name. It's probably wise also
+to associate your program with a run configuration as follows. Click on the gray down arrow near the green "play" triangle
+near the top center of the window and select `Edit Configurations...`, click on the `+` button in the top left corner,
+select `Application`, give your configuration a `Name` (typically the same as the name of the class you've just chosen),
+type the name of your class again in the `Main class` field, and click `OK`.
```

Comment:
I never bother with this step. I believe a new configuration is created when you first run the file, and the default configuration works fine.

Reply:
I thought I was running into issues if I didn't explicitly created configurations.  But I wasn't able to repeat these behaviors.  I didn't realize IntelliJ was creating them automatically when clicking on the "play" button near the class name.  I'll drop this change.

Reply:
Based on your comments and my improved understanding of IntelliJ I've decided to abandoned these changes.